example/src/examples/ScrollableExample/ScrollableExampleFirst.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
```

Comment:
Why is this file in the example folder?

Reply:
Yes, you are right, I am removing that file from example folder now.